cocos2dx/actions/CCActionEase.h

```diff
@@ -407,7 +407,7 @@ class CC_DLL CCEaseElasticInOut : public CCEaseElastic
 class CC_DLL CCEaseBounce : public CCActionEase
 {
 public:
-    float bounceTime(float time);
+    static float bounceTime(float time);
```

Comment:
This function can be static , so changed to static.
Does this has back compatiability porblem?